frontend/src/scenes/plugins/plugin/PluginImage.tsx

```diff
@@ -6,18 +6,22 @@ import { PluginInstallationType } from 'scenes/plugins/types'
 
 export function PluginImage({
     url,
+    icon,
     pluginType,
     size = 'medium',
 }: {
     url?: string
+    icon?: string
     pluginType?: PluginInstallationType
     size?: 'medium' | 'large'
 }): JSX.Element {
     const [state, setState] = useState({ image: imgPluginDefault })
     const pixelSize = size === 'large' ? 100 : 60
 
     useEffect(() => {
-        if (url?.includes('github.com')) {
+        if (icon) {
+            setState((state) => ({ ...state, image: icon }))
+        } else if (url?.includes('github.com')) {
             const { user, repo } = parseGithubRepoURL(url)
             setState({ ...state, image: `https://raw.githubusercontent.com/${user}/${repo}/main/logo.png` })
```

Comment:
If the `icon` is included it'll work, but this broke down for me when looking for `logo.png` if the repo url contained a folder. Should it be included here too?

Reply:
Yup, I believe I actually fixed this as part of #12308 if I'm not mistaken so probably better to merge that first and then incorporate here

Reply:
Merged the fix from #12308 now that that's merged, will give this a test